test/org/sagebionetworks/bridge/validators/SurveySaveValidatorTest.java

```diff
@@ -881,11 +881,11 @@ public void operatorsInInfoScreenRulesAreValidated() {
         info.setPromptDetail("prompt detail");
         info.setIdentifier("identifier");
         info.setGuid("guid");
-        info.setRules(Lists.newArrayList(
+        info.setAfterRules(Lists.newArrayList(
                 new SurveyRule.Builder().withValue("foo").withOperator(Operator.EQ).withEndSurvey(true).build()));
```

Comment:
Should we also add beforeRules to verify that both are being tested? (Whether this logically goes here or in beforeRulesAreValidated() is up to you.)